src/stories/FormFields.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Props, Story, Preview } from '@storybook/addon-docs/blocks';
+import { action } from "@storybook/addon-actions";
+import { withKnobs, object, boolean, text, select } from '@storybook/addon-knobs';
+import { withInfo } from 'storybook-addon-vue-info'
+import TextField from '../components/TextField.vue';
+import ChecDropdown from '../components/ChecDropdown.vue';
+
+<Meta title="Form fields" component={TextField} />
```

Comment:
Did you deliberately put this into its own "Overview" story? If so, all good.

The text field's label says "dropdown" which confused me.

Reply:
Yep I did. I've renamed the field name. I forgot to change it back after I was making sure the fields were consistent.

Reply:
The text field label should still be updated